iotedgehubdev/composeproject.py

```diff
@@ -167,10 +167,17 @@ def config_env(self, env_list, env_section):
 
     def parse_routes(self):
         routes = self.module_content['$edgeHub']['properties.desired']['routes']
+        schema_version = self.module_content['$edgeHub']['properties.desired']['schemaVersion']
         routes_env = []
         route_id = 1
-        for path in routes.values():
-            routes_env.append('routes__r{0}={1}'.format(route_id, path))
+
+        for route in routes.values():
+            if isinstance(route, string_types):
+                routes_env.append('routes__r{0}={1}'.format(route_id, route))
+            else:
+                if schema_version == "1.0":
```

Comment:
we need check the version >= 1.1 instead equal 1.0

Reply:
OK